### DISCUSSION TITLE
纽约时代广场 大屏幕广告 2分钟 播一次40刀

### DISCUSSION THREAD

**Discussion by frshgrass** [WeNeedHome/SummaryOfLoanSuspension#1202] [General]
大家实在走投无路，可以走这条。共产威权体制最爱面子，苏联就是例子。 

活人千万不能被尿憋死！！！

**Comment by Cegazhos** [comment]
你在外面叫得再欢也没用，大陆脑残被洗完脑后首先看不见第二也不敢说什么。
在大陆，活人就是别憋死又如何。
